great_expectations/datasource/fluent/sources.py

```diff
@@ -625,6 +637,11 @@ def __getattr__(self, attr_name: str):
             elif crud_method_type == CrudMethodType.ADD_OR_UPDATE:
                 return self.create_add_or_update_crud_method(datasource_type, docstring)
             elif crud_method_type == CrudMethodType.DELETE:
+                # deprecated-v0.17.2
+                warnings.warn(
+                    f"`{attr_name}` is deprecated as of v0.17.2 and will be removed in v0.19. Please use `delete_datasource` moving forward.",
+                    DeprecationWarning,
+                )
```

Comment:
Why does this not emit anything to stdout/stderr??? 🤔

Reply:
no luck setting `stacklevel=2`

Reply:
Is it possible that this is being captured by [catch_warnings](https://docs.python.org/3/library/warnings.html#warnings.catch_warnings) later in the stack?

Reply:
seems to be good with the tests I've written!